packages/dds/shared-object-base/src/types.ts

```diff
@@ -8,6 +8,18 @@ import { IChannel } from "@fluidframework/datastore-definitions";
 import { ISequencedDocumentMessage } from "@fluidframework/protocol-definitions";
 import { IGarbageCollectionData } from "@fluidframework/runtime-definitions";
 
+/**
+ * Events emitted by {@link ISharedObject}.
+ *
+ * @remarks
+ * ### "pre-op"
+ *
+ * TODO
+ *
+ * ### "op"
```

Comment:
We should generally discourage use of the "op" event, it is generally better for consumers to use the DDS-specific events that are exposed.  Ops should be considered an internal implementation detail of the DDS rather than a public interface.  The "op" event fires just after applying an incoming op to a shared object.